apps/admin_api/lib/admin_api/v1/controllers/mint_controller.ex

```diff
@@ -0,0 +1,44 @@
+defmodule AdminAPI.V1.MintController do
+  @moduledoc """
+  The controller to serve mint endpoints.
+  """
+  use AdminAPI, :controller
+  import AdminAPI.V1.ErrorHandler
+  alias EWallet.Web.{SortParser, Paginator, Preloader}
+  alias EWalletDB.{Token, Mint}
+  alias Plug.Conn
+
+  @mapped_fields %{
+    "created_at" => "inserted_at"
+  }
+  @sort_fields [:id, :description, :amount, :confirmed, :inserted_at, :updated_at]
+  @preload_fields [:token, :account, :transfer]
+
+  @doc """
+  Retrieves a list of mints.
+  """
+  @spec all_for_token(Conn.t(), map() | nil) :: map()
```

Comment:
`respond_multiple/2` and `handle_error/2` both return `Conn.t()` I think. Suggesting...

```ex
@spec all_for_token(Conn.t(), map() | nil) :: Conn.t()
```